base/bucket_gocb.go

```diff
@@ -1116,12 +1119,16 @@ func (bucket *CouchbaseBucketGoCB) WriteCasWithXattr(k string, xattrKey string,
 		if cas == 0 {
 			// TODO: Once that's fixed, need to wrap w/ retry handling
 			gocbExpvars.Add("WriteCasWithXattr_Insert", 1)
-			docFragment, err := bucket.Bucket.MutateInEx(k, gocb.SubdocDocFlagReplaceDoc, 0, exp).
-				UpsertEx(xattrKey, xv, gocb.SubdocFlagXattr).                                                               // Update the xattr
-				UpsertEx(xattrCasProperty, "${Mutation.CAS}", gocb.SubdocFlagXattr|gocb.SubdocFlagUseMacros).               // Stamp the cas on the xattr
-				UpsertEx(xattrBodyHashProperty, "${Mutation.value_crc32c}", gocb.SubdocFlagXattr|gocb.SubdocFlagUseMacros). // Stamp the body hash on the xattr
-				UpsertEx("", v, gocb.SubdocFlagNone).                                                                       // Update the document body
-				Execute()
+
+			mutateInBuilder := bucket.Bucket.MutateInEx(k, gocb.SubdocDocFlagReplaceDoc, 0, exp).
+				UpsertEx(xattrKey, xv, gocb.SubdocFlagXattr).                                                // Update the xattr
+				UpsertEx(xattrCasProperty, "${Mutation.CAS}", gocb.SubdocFlagXattr|gocb.SubdocFlagUseMacros) // Stamp the cas on the xattr
+			if crc32cMacroExpansionSupported {
+				mutateInBuilder.UpsertEx(xattrBodyHashProperty, "${Mutation.value_crc32c}", gocb.SubdocFlagXattr|gocb.SubdocFlagUseMacros) // Stamp the body hash on the xattr
+			}
+			docFragment, err := mutateInBuilder.UpsertEx("", v, gocb.SubdocFlagNone). // Update the document body
+													Execute()
```

Comment:
Is there something weird going on with the whitespace here, or is that a github UI artifact?

Reply:
I moved the `.Execute()` statements to their own lines, which should fix this and improve overall readability.